### ISSUE TITLE
Basis set for non-normal distribution

### ISSUE THREAD

**Issue by jslefche** [jslefche/piecewiseSEM#57]
The order of the models (aka, the order in which variables first appear in the model) affect the outcome of significance tests for non-normal distributions. 

E.g., 

```
data = data.frame(
  a = sample(1:100, 100, replace = T),
  b = sample(1:100, 100, replace = T),
  c = sample(1:100, 100, replace = T),
  d = sample(1:100, 100, replace = T)
)

modelList = list(
  glm(c ~ a, "poisson", data),
  glm(b ~ a, "poisson", data),
  glm(d ~ c + b, "poisson", data)
)

modelList2 = list(
  glm(b ~ a, "poisson", data),
  glm(c ~ a, "poisson", data),
  glm(d ~ c + b, "poisson", data)
)

sem.basis.set(modelList)
sem.basis.set(modelList2)

sem.fit(modelList, data)
sem.fit(modelList2, data)

```

Switching this to `lm` yields identical goodness-of-fit results.

**Comment by jslefche** [comment]
Put more simply, `y` and `x` are not interchangeable in GLMs. So what to do about the basis set when direction of causality is unclear (e.g., b -> c or c->b in above, since both are endogenous variables)

```
summary(glm(b ~ c, "poisson", data))$coefficients
summary(glm(c ~ b, "poisson", data))$coefficients
```

**Comment by jebyrnes** [comment]
Perhaps LR tests using type II comparisons?

**Comment by jslefche** [comment]
Fixed in 2.0
